runtime/lib/backends/nccl/providers/op_registration.cc

```diff
@@ -26,16 +30,72 @@ namespace cuda {
 
 void RegisterNCCLOps(KernelRegistry *registry) {
   registry->Register(
-      "NCCLRecv_f32",
+      "nccl.Recv",
```

Comment:
I think @qingyunqu suggested using "nccl::" prefix. Is there a problem using that prefix?

Reply:
Yep, "nccl::" let the parser parse it as  SymbolRefAttr, not FlatSymbolRefAttr. To make it correct, we must change the definition of "Byre Dialect", it will involve other efforts.